schema/v1.1.0/dataset_config_models_extended.py

```diff
@@ -101,6 +102,33 @@
 running_network_validation = False
 
 
+# ==============================================================================
+# Helper Functions
+# ==============================================================================
+def check_excludelist_skip_validation(obj: BaseModel, field_name: str, case_sensitive: bool = False) -> bool:
```

Comment:
nit: Thoughts on naming the methods `check_skip_validation`?

Reply:
will do!